src/content/documentation/enterprise/enterprise-policies-that-impact-extensions.md

```diff
@@ -60,15 +60,15 @@ These policies control whether users can install, disable, or remove web extensi
 
 These policies may affect the behavior of third-party extensions because of the Firefox features these policies disable or restrict.
 
-- **DisableDeveloperTools** — This policy removes access to all developer tools. With this policy in place, it limits users’ ability to create extensions effectively.
+- **`DisableDeveloperTools`** — This policy removes access to all developer tools. With this policy in place, it limits users’ ability to create extensions effectively.
 
-- **DisableFirefoxAccounts** — This policy disables Sync. With this policy in place, third-party extensions that make use of the sync functionality may not work correctly.
+- **`DisableFirefoxAccounts`** — This policy disables Sync. With this policy in place, third-party extensions that make use of the sync functionality may not work correctly.
```

Comment:
I'd suggest removing the bold now it's in backticks.

Reply:
(After a git kerfuffle) Fixed in c81225b.